### PR TITLE
feat(loadtest): allow rerunning on the same state

### PR DESCRIPTION
**Pull request by jakmeier**
Previously the second run on the same state would fail due to the
funding accounts already existing.

Now one can choose between reusing old state (default) or starting
a new run with a new state (set a `--social-db-index` to a unique value)

Already existing accounts are simply not created, so things should work
fine as long as the initial balance per account doesn't run out.

### CODE REVIEW

File: pytest/tests/loadtest/locust/common/social.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def on_locust_init(environment, **kwargs):
             )
 
         social_contract_code = environment.parsed_options.social_db_wasm
-        contract_key = key.Key.from_random(environment.social_account_id)
+        contract_key = key.Key.from_seed_testonly(environment.social_account_id)
```
Comment (akashin) [inline pytest/tests/loadtest/locust/common/social.py:294]: I believe you need two arguments for `Key.from_seed_testonly` [1]

[1] https://github.com/near/nearcore/blob/385e9f1f8ed1ec00cc5128fefb183c975b60ede0/pytest/lib/key.py#L41

**Review by akashin — APPROVED** [review]

File: pytest/tests/loadtest/locust/common/social.py (hunk 1 above)
Comment (jakmeier) [reply pytest/tests/loadtest/locust/common/social.py]: Good catch! I lost that one while untangling ~10 commits into 4 PRs, I hope it's the only mistake I made...

(Also, we seem to always use the same values for both args, maybe we should refactor to simplify?)

File: pytest/tests/loadtest/locust/common/base.py [hunk 2]
```diff
@@ -251,6 +255,9 @@ def poll_tx_result(self, meta, params):
             meta["response"] = result_response.content
             raise
 
+    def account_exists(self, account_id: str) -> bool:
+        return not "error" in self.node.get_account(account_id, do_assert=False)
```
[the remark this answers is not in the document]
Comment (jakmeier) [reply pytest/tests/loadtest/locust/common/base.py]: changed in this instance (there are several others already merged where we could change it, too)

File: pytest/tests/loadtest/locust/common/social.py (hunk 1 above)
Comment (akashin) [reply pytest/tests/loadtest/locust/common/social.py]: Agreed. I can imagine a use case where one would require to customize a seed, but so far we've been fine with using the same default seed value. We can either add another method `from_default_seed` or just add a default value to the `seed` argument in this function.

Comment (jakmeier) [reply pytest/tests/loadtest/locust/common/social.py]: #9234

**Pull request merged**